src/webargs/core.py

```diff
@@ -34,6 +32,12 @@
 # generic type var with no particular meaning
 T = typing.TypeVar("T")
 
+# a set of fields which are known to satisfy the `is_multiple` criteria, but
+# which come from marshmallow and therefore don't know about webargs (and
+# do not set `is_multiple=True`)
+# TODO: should `ma.fields.Tuple` be added here?
+KNOWN_MULTI_FIELDS: typing.Tuple[typing.Type, ...] = (ma.fields.List,)
```

Comment:
Should this be mutable to allow users to add field classes here?

Reply:
I didn't really think about this var as part of the public interface. I'm not against it, I just didn't think of that at all! 😄

If we start going down the path of it being customizable I might want to move it onto the parser so that it can be controlled by subclassing. Right now pretty much all configurable behavior can be set on a parser. Should I take a look at doing that?

Reply:
Sure. Good point.

But no big deal if we keep it as a global. It's not like someone will be instantiating several parsers with different `KNOWN_MULTi_FIELD` values.

Reply:
I think you're right that multiple parsers with different values is an unlikely/strange use-case.

It's getting late in my local time (US Eastern), so I'll come back to this tomorrow. But right now I'm inclined to change it to be a list and merge the result. At least that way, it's not actively stopping anyone from modifying it.

Reply:
Yeah, it is not a really friendly API, but it doesn't get in the way of advanced users.